src/App/Pages/Accounts/TwoFactorPageViewModel.cs

```diff
@@ -245,28 +267,32 @@ private async Task DuoFramelessAuthenticateAsync()
                 return;
             }
 
-            string response = null;
-            if (authResult != null && authResult.Properties.TryGetValue("code", out var resultData))
+            await _deviceActionService.HideLoadingAsync();
```

Comment:
🤔 How does it feel from a user perspective? Because doing it like this, the loading will disappear and after a brief time it'll appear again on the screen. Maybe it's happening too fast so it's not a big deal but wanted to ask just in case.
If it's noticeable and an issue (I'd ask product just in case), then I'd only hide the loading just before showing another dialog that we know the flow stops there so the user won't notice anything weird about it. Furthermore, you could call `SubmitAsync(false)` again, and check inside whether a loading is already being presented to hide it before displaying a another dialog.

Reply:
https://github.com/bitwarden/mobile/assets/4648522/50a6e218-c974-4ec6-9442-ca8779de6ba3

It happens so fast, I don't think that will be an issue.